src/test/java/uk/ac/cam/cl/dtg/teaching/pottery/controllers/TestEnvironment.java

```diff
@@ -46,19 +44,11 @@
 import uk.ac.cam.cl.dtg.teaching.pottery.exceptions.RetiredTaskException;
 import uk.ac.cam.cl.dtg.teaching.pottery.exceptions.TaskNotFoundException;
 import uk.ac.cam.cl.dtg.teaching.pottery.exceptions.TaskStorageException;
-import uk.ac.cam.cl.dtg.teaching.pottery.model.Action;
-import uk.ac.cam.cl.dtg.teaching.pottery.model.Execution;
-import uk.ac.cam.cl.dtg.teaching.pottery.model.Step;
 import uk.ac.cam.cl.dtg.teaching.pottery.model.TaskInfo;
-import uk.ac.cam.cl.dtg.teaching.pottery.model.Testcase;
+import uk.ac.cam.cl.dtg.teaching.pottery.task.*;
```

Comment:
please fully expand imports

Reply:
Expanded. I think I've convinced intelliJ to stop wildcarding them when there are several.